blaze-server/src/main/scala/org/http4s/server/blaze/BlazeServerBuilder.scala

```diff
@@ -150,20 +155,75 @@ class BlazeServerBuilder[F[_]](
       maxHeadersLen: Int = maxHeadersLen): Self =
     copy(maxRequestLineLen = maxRequestLineLen, maxHeadersLen = maxHeadersLen)
 
+  @deprecated("Use withSSLContext and withTLSParameters instead", "0.21.0-RC3")
```

Comment:
Translating this call is a drag (note the heft of the implementation), but we're not trying to be a general SSL configuration library.

If people really want something like this, we should put it in a function that returns an `F[(SSLContext, SSLParameters)]`, and let them flatMap it in.  Then we can get rid of the nasty effect in this builder.